src/main/scala/nl/biopet/bioconda/BiocondaUtils.scala

```diff
@@ -0,0 +1,209 @@
+
+    // Without "--no-color" scala doesn't match the strings properly. Color matters in string comparison!
+    val branchList: Array[String] = {
+      if (remotes) {
+        git.apply("branch", "-a", "--no-color")(repo, log).split("\\n")
+      } else {
+        git.apply("branch", "--no-color")(repo, log).split("\\n")
+      }
+    }
+    val branches = new ListBuffer[String]
+
+    // For each branch
+    // Remove that annonoying *
+    // Split on / and get the last item(remotes/origin/branch) => branch
+    // Trim away all spaces
+    branchList.foreach(x =>
+      branches.append(x.replaceFirst("\\*", "").split("/").last.trim()))
+    branches.toList.contains(branch)
+
+  }
+  def getVersionFromYaml(metaYaml: File): String = {
+    val versionRegex: Regex = "version:.([0-9\\.]+)".r
+    val yaml = Source.fromFile(metaYaml).getLines().mkString
+    val matches = versionRegex.findAllIn(yaml).matchData
+    if (matches.isEmpty) {
+      throw new Exception(s"No version found in: ${metaYaml.getPath}")
+    }
+    val version = matches.toList.head.group(1)
+    version
+  }
+
+  def dockerInstalled(log: ManagedLogger, path: Option[String] = None): Unit = {
+    val testCommand = "docker version -f '{{.Client.Version}}'"
+    def test: ProcessBuilder =
+      if (path.isDefined) {
+        Process(Seq("bash", "-c", testCommand),
+                None,
+                "PATH" -> path.getOrElse("$PATH"))
+      } else {
+        Process(Seq("bash", "-c", testCommand), None)
+      }
+    try { test.!!(log) } catch {
+      case e: Exception =>
+        throw new Exception(s"Docker does not run: ${e.getMessage}")
+    }
+  }
+  def circleCiCommand(cwd: File, args: Seq[String], log: ManagedLogger) = {
+    val path = cwd.getPath
+    val command = Seq("docker",
+                      "run",
+                      "--rm",
+                      "-v",
+                      "/var/run/docker.sock:/var/run/docker.sock",
+                      "-v",
+                      s"$path:$path",
+                      "--workdir",
+                      s"$path",
+                      "circleci/picard",
+                      "circleci") ++ args
+    (Process(command, cwd).lineStream(log)).foreach(line => log.info(line))
+    // if (exit != 0) throw new Exception(s"Command ${command.mkString(" ")} failed with exit code: ${exit}.")
+  }
+
+  def pullLatestUtils(log: ManagedLogger) = {
+    Process(Seq("docker", "pull", "bioconda/bioconda-utils-build-env"))
+      .lineStream(log)
+      .foreach(line => log.info(line))
+  }
+
+  def copyDirectory(source: File,
+                    dest: File,
+                    permissions: Boolean = true): Unit = {
+    assert(source.isDirectory, "Source should be a directory")
+    if (dest.exists()) {
+      if (!dest.isDirectory) {
+        throw new IOException(
+          s"Destination ${dest.getAbsolutePath} is a file, not a directory.")
+      }
+    } else {
+      dest.mkdirs()
```

Comment:
`dest.mkdirs` is like mkdir -p, is this what you want? If you mean just mkdir then it should be `dest.mkdir`